PurchasesTests/Purchasing/PurchasesDeprecation.swift

```diff
@@ -15,6 +15,7 @@ import Foundation
 import RevenueCat
 
 // Protocol that enables us to call deprecated methods without triggering warnings.
+// swiftlint:disable missing_docs
```

Comment:
Let's make the `protocol` `internal` instead. Nobody consumes types in tests so it's pointless 😋

Reply:
sure! 🙃